data.sql

```diff
@@ -1,13 +1,17 @@
 /* Populate database with sample data. */
 
-insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
-values ('Agumon', '2020-02-03', 0, true, 10.23);
-
 insert into animals 
-values (DEFAULT, 'Gabumon', '2018-11-15', 2, true, 8);
-
+values (DEFAULT, 'Charmander', '2020-02-08', 0, false, 11);
 insert into animals 
-values (DEFAULT, 'Pikachu', '2021-01-07', 1, false, 15.04);
-
+values (DEFAULT, 'Plantmon', '2021-11-15', 2, true, 5.7);
+insert into animals 
+values (DEFAULT, 'Squirtle', '1993-04-02', 3, false, 12.13);
 insert into animals 
-values (DEFAULT, 'Devimon', '2017-05-12', 5, true, 11);
+values (DEFAULT, 'Angemon', '2005-06-12', 1, true, 45);
```

Comment:
- [ ] Please make sure to insert the correct values so the queries select the correct data. The `weight_kg` of the first 4 animals should be inserted as a minus so the first insertion should look like the following:
```SQL
insert into animals 
values (DEFAULT, 'Charmander', '2020-02-08', 0, false, -11);
```